### ISSUE TITLE
i've been put in smurf pool

### ISSUE THREAD

**Issue by Deni-Lira** [ValveSoftware/Dota2-Gameplay#13078]
### Description

i finished my 100 hour to start playing rankeds, and won a lot of matchs in a row, now i cant find queues no matter how many time i wait, i have just a few games but im not a smurf. what can i do? i'm not a smurf and now i cant play just because i won a lot of games

**Comment by Hairraiserz** [comment]
Thanks for sharing this. [best salon in mohali](https://hairraiserzluxesunny125.business.site/)

**Comment by TobyFlendersonFromHR** [comment]
I agree, check out the salon and you won't be in smurf pool anymore!

**Comment by antonio-dropulic** [comment]
I have the same experience. I have written to steam support, but no help was provided and the ticket closed. They say this is a developer issue, and it does feel that way.

I'm a 10+ years on and off player that's been hovering around ancient since the ranked system has been introduced. Last year I had steam support delete my account so because i was having trouble autoregulating my playtime. Now when I can play the game I so much love I have gone through the 100 hours to play ranked again, and ended in a smurf pool. Queue times are terrible (20+ minutes), and game quality is surprisingly bad. 

I assume this small pool of players has different mmr/beh ratings than the normal population. As soon as my games got players with normal number of games(average was thousands instead of hundreds) i started winning way more often and ended in the pool again. This emergent behavior should be fixed for legitimate new accounts, and/or there should be a way to validate your account somehow and worst case manually remove it from the pool. 

I'm really disappointed in the lack of constructive feedback from the support team. I'm sure this kinda ranked behavior is not limited to my use case which may happen seldom. This can potentially affect players transitioning from other MOBAS or players that learn the game in LAN (this was once the primary way to learn the game). 

Feedback would be really appreciated. Help even more so. I can speak from my own case this is making me contemplate account buying, and i really don't want to resort to that.
